backend/run_scrapers.py

```diff
@@ -47,6 +53,34 @@
     # Write list of welsh groups to csv
     write_data_to_CSV(welsh_groups[0], welsh_groups[1], fn_groups_cleaned)
 
+    # Count the number of groups per area to produce the groupCount layer
+
+    # Get boundary shapes and names for la and lsoa levels
+    boundary_info_LA = get_boundaries_LA(
+        "backend/data/static/geoboundaries/boundaries_LA.geojson"
+    )
+    boundary_info_LSOA = get_boundaries_LSOA(
+        "backend/data/static/geoboundaries/boundaries_LSOA.geojson"
+    )
+
+    # Make seperate counts of groups per la and lsoa
+    count_LA = count_groups(
+        "backend/data/live/cleaned/groups.csv", boundary_info_LA, "lad18cd"
```

Comment:
Going to put an issue in about the geographies files too so we don't forget! Particularly because I've just realised I'm using the 2019 boundaries, and yours are the 2018 ones !